src/llm_api/_client.py

```
@@ -81,7 +90,7 @@ def __init__(
         if base_url is None:
             base_url = os.environ.get("LLM_API_BASE_URL")
         if base_url is None:
-            base_url = f"https://api.example.com"
+            base_url = f"http://localhost:8000/openai/v1"
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)